config/sbnd/sbnd_full_chain_250328.cfg

```
@@ -501,7 +501,7 @@ post:
   calo_ke:
     run_mode: both
     scaling: 1.
-    shower_fudge: 1/0.77 # previous:  1/0.83
+    shower_fudge: 1/0.763 # previous:  1/0.77
```

```suggestion
    shower_fudge: 1/0.763
```
This also needs to be checked in new studies.